proxyui/src/main/webapp/resources/js/webjars.include

```diff
@@ -38,11 +38,12 @@
 <!----------- mousetrap ------------>
 <script src="<c:url value="/webjars/mousetrap/1.6.0/mousetrap.min.js" />" type="text/javascript"></script>
 
-<!--------- user resources --------->
+<!----------- downloadjs ----------->
+<script src="<c:url value="/webjars/downloadjs/1.4.8/download.min.js" />" type="text/javascript"></script>
```

Comment:
NB this is different than the manually included `download.min.js`. But the functionality is the same.